### PR TITLE
Add some sprite stuff

### PR DESCRIPTION
**Pull request by wipfli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,12 @@ and examples at [eodagmbh.github.io/py-maplibregl/examples/](https://eodagmbh.gi
 - [MapBlockly](https://mapblockly.github.io/) - MapBlockly is a simple and fun way to learn and build Map with Blockly using MapLibre.
 - [MapInventor](https://mapinventor.github.io/) - MapInventor is the visual language built on top of MapBlockly.
 
+## Sprites
```
Comment (HarelM) [inline README.md:73]: It's probably worth adding ekastic's spritezero, and maybe update the sprite spec documentation with tool to create sorites for maolibre.

**Review by HarelM — APPROVED** [review]

**Comment by josxha** [comment]
Just stumbled upon spreet from the MapLibre style specs. Is there a reason isn't merged yet?

**Comment by HarelM** [comment]
For no good reason, we should probably update the style spec docs on tools that are actually maintained and can generate sprites, I'm not familiar enough with those and haven't been generating sprites lately...
Feel free to open a PR against the style-spec-repo sprite.md file to update the suggestion there...

**Comment by wipfli** [comment]
Ah I just forgot to press the merge button. Thanks for the ping @josxha

**Pull request merged**